frontend/src/components/ContentFooter.spec.js

```diff
@@ -33,7 +33,7 @@ describe('ContentFooter', () => {
       })
 
       it('renders the copyright year', () => {
-        expect(mocks.$t).toBeCalledWith('footer.copyright.year', { year: 2022 })
+        expect(mocks.$t).toBeCalledWith('footer.copyright.year', { year: 2023 })
```

Comment:
I fixed this in https://github.com/gradido/gradido/pull/2505

Reply:
Solved there